app/src/main/res/layout-sw600dp-land/admin_pin_activity.xml

```diff
@@ -0,0 +1,103 @@
+<?xml version="1.0" encoding="utf-8"?>
+<layout xmlns:android="http://schemas.android.com/apk/res/android"
+  xmlns:app="http://schemas.android.com/apk/res-auto"
+  xmlns:profile="http://schemas.android.com/tools">
+
+  <data>
+
+    <variable
+      name="viewModel"
+      type="org.oppia.android.app.profile.AdminPinViewModel" />
+  </data>
+
+  <androidx.core.widget.NestedScrollView
+    android:id="@+id/scrollViewAdminPin"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:fillViewport="true">
+
+    <androidx.constraintlayout.widget.ConstraintLayout
+      android:id="@+id/admin_auth_container"
+      android:layout_width="match_parent"
+      android:layout_height="match_parent"
+      android:background="@color/oppiaBackgroundYellowIvory"
+      android:clipToPadding="false"
+      android:paddingStart="232dp"
+      android:paddingEnd="232dp"
```

Comment:
Similar to this comment, you should apply the concepts here too. https://github.com/oppia/oppia-android/pull/2124#pullrequestreview-533595345

Reply:
do I have to reformat it?

Reply:
@rt4914 can you specify which concepts
is it negative margin/padding or reformatting code

Reply:
Done